finals-2024-01-15/code/5094-viktor-lozanovski/src/app/movie-list/movie-services.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Movie, Student } from '../models/student';
+import { Observable } from 'rxjs';
+
+
+const BASE_URL = 'http://localhost:3000/';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class MoviesService {
+
+  constructor(private http: HttpClient) { }
+
+  getMovies() {
```

Comment:
The code in the service is ok, but it's not actually used anywhere. +5 points for the service design, and +5 points for injecting and using `HttpClient`